spice-carla-sync-service/CarlaUtils.cs

```diff
@@ -190,7 +282,7 @@ private bool SendSPDEmail(string attachmentContent, string attachmentName)
 
             using (var stream = new MemoryStream())
             using (var writer = new StreamWriter(stream))    // using UTF-8 encoding by default
-            using (var mailClient = new SmtpClient(Configuration["SMTP_HOST"]))
+            using (var mailClient = new SmtpClient(Configuration["SMTP_HOST"], 2525))
```

Comment:
This appears to be local development code.  Can you put the port in the SMTP HOST?